.github/workflows/stale.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         days-before-pr-stale: 30
         days-before-pr-close: 180
         operations-per-run: 25
-        start-date: '2020-12-01T00:00:00Z'
+        start-date: '2018-10-01T00:00:00Z'
```

Comment:
can commit separately

Reply:
OK, separate it into `github-ci` pr later